source/linux/Foundational_Components/Kernel/Kernel_Drivers/Network/CPSWng.rst

```diff
@@ -44,11 +54,23 @@ configurations:
     The two configurations are mutually exclusive. Only one of the two
     can be enabled for a build.
 
-.. note::
-    | Enabling the CPSW Proxy Client configuration or the CPSWng Native Ethernet configuration
-    | requires applying a Linux device-tree overlay at U-Boot using the **name_overlays** U-Boot environment variable.
-    | For details regarding applying CPSW Proxy Client overlay, refer :ref:`cpsw_proxy_client_enable`.
-    | For details regarding applying CPSWng Native Ethernet overlay, refer :ref:`cpswng_native_eth_overlays`.
+.. ifconfig:: CONFIG_part_family in ('J7_family')
+
+   .. note::
+
+      | Enabling the CPSW Proxy Client configuration or the CPSWng Native Ethernet configuration
+      | requires applying a Linux device-tree overlay at U-Boot using the **name_overlays** U-Boot environment variable.
+      | For details regarding applying CPSW Proxy Client overlay, refer :ref:`cpsw_proxy_client_enable`.
+      | For details regarding applying CPSWng Native Ethernet overlay, refer :ref:`cpswng_native_eth_overlays`.
+
+
+.. ifconfig:: CONFIG_part_family in ('AM62PX_family')
+
+   .. note::
+
+      | Enabling the CPSW Proxy Client configuration requires applying a Linux device-tree overlay
+      | at U-Boot using the **name_overlays** U-Boot environment variable.
+      | For details regarding applying CPSW Proxy Client overlay, refer :ref:`cpsw_proxy_client_enable`.
```

Comment:
In the future do not use `|` if you can avoid it. I've often see it break directives by incorrectly eating newline characters.